### DISCUSSION TITLE
Coverage map

### DISCUSSION THREAD

**Discussion by Advaith2212** [NVlabs/sionna#238] [Q&A] [answered]
Hi sir,
I would like to understand the dependency on the `scene.frequency` for coverage map generation. 

Initially in the example code, `scene.frequency = 2.4e9  ` is set and coverage is computed.

When I change it to a different value, the coverage map remains the same. 

So would like to understand this please

**Comment by jhoydis** [comment]
Did you recompute the coverage map after changing the frequency?

**Reply by Advaith2212** [reply]
Ah yes sir , 
The steps I followed: 
        1. set `fc`
        2. Add Transmitter and receiver to scene 
        3. Compute path 
        4. Compute coverage map 
       
 I restarted the setup and set `fc = 30e6` and repeated the process. Did I miss any step in this ?

**Reply by Advaith2212** [reply]
### Covereage map for `fc = 30Ghz`
![coverage_60Ghz](https://github.com/NVlabs/sionna/assets/64831500/ef01c97b-af1c-425e-9b73-010dcf06d46e)

### Coverage for `fc = 3.5Ghz` 
![coverage_3 5Ghz](https://github.com/NVlabs/sionna/assets/64831500/50a464bd-ba74-49e3-ad50-3f91c323717b)

**Reply by jhoydis** [reply]
Could you provide the ``coverage_map.show()`` results that show also the colorbar?

**Reply by Advaith2212** [reply]
### `fc = 3.5Ghz`
![3 5Ghz](https://github.com/NVlabs/sionna/assets/64831500/a3a96792-7550-4fd1-9aed-41c3f858b478)

### `fc = 30Ghz`
![30Ghz](https://github.com/NVlabs/sionna/assets/64831500/cf836352-9cbc-4805-a522-5e8ac9eb6571)


P.S :
1. `cm.show()` displays the coverge map two times in the output window. Is that supposed to be like that sir ? 

2. For `fc > 30e9` the `scene.compute_paths()` function isn't working. Is it because some of the radio materials dont support the frequency.

![itu_brick](https://github.com/NVlabs/sionna/assets/64831500/b8f57124-b077-421f-a53a-523adc1316b0)

**Reply by jhoydis** [reply, marked as answer]
As you can see, the coverage maps are not identical. The pathloss is much larger for 30GHz as it should be.
You can just put a ";" behind ``cm.show();``. This is due to the behaviour of Jupyter and not a bug of Sionna.
Concerning higher frequencies, you would need to change the material properties. Some ITU materials are simply not defined for all frequency ranges.
